components/nimbus/ios/Nimbus/Nimbus.swift

```diff
@@ -75,6 +75,10 @@ extension Nimbus: FeaturesInterface {
         }
     }
 
+    public func recordEvent(eventId: String) {
```

Comment:
Same thing as above, this should be part of the general Nimbus API and not the FeatureInterface. Either move this to the `public class Nimbus` definition or into an appropriate extension.